.travis-setup.sh

```diff
@@ -12,29 +12,41 @@ fetch_stack_osx() {
 }
 
 fetch_stack_linux() {
-  curl -sL https://www.stackage.org/stack/linux-x86_64 | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack';
+  curl -sL https://www.stackage.org/stack/linux-x86_64 | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
```

Comment:
These changes were needed so that the BLAS fortran library could be built from source.   fetch_blas_x() downloads and unzips the source code.